diff --git a/data/backend/boltz.conf b/data/backend/boltz.conf
index 894e19d..c790978 100755
--- a/data/backend/boltz.conf
+++ b/data/backend/boltz.conf
@@ -36,11 +36,11 @@ maxSwapAmount = 40_294_967
 minSwapAmount = 50_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 300
+  reverse = 300
+  swapTaproot = 300
+  swapMinimal = 300
+  swapMaximal = 350
 
 [[pairs]]
 base = "L-BTC"
@@ -52,11 +52,11 @@ maxSwapAmount = 40_294_967
 minSwapAmount = 50_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 300
+  reverse = 300
+  swapTaproot = 300
+  swapMinimal = 300
+  swapMaximal = 350
 
 [[pairs]]
 base = "RBTC"
