diff --git a/data/backend/boltz.conf b/data/backend/boltz.conf
index 247da7e..38991ea 100755
--- a/data/backend/boltz.conf
+++ b/data/backend/boltz.conf
@@ -112,14 +112,6 @@ maxZeroConfAmount = 0
     privateKeyPath = "/root/.lightning/regtest/hold/client-key.pem"
     certChainPath = "/root/.lightning/regtest/hold/client.pem"
 
-  [currencies.cln.mpay]
-    host = "cln-2"
-    port = 9293
-
-    rootCertPath = "/root/.lightning/regtest/mpay/ca.pem"
-    privateKeyPath = "/root/.lightning/regtest/mpay/client-key.pem"
-    certChainPath = "/root/.lightning/regtest/mpay/client.pem"
-
 
 [liquid]
 symbol = "L-BTC"
diff --git a/docker-compose.yml b/docker-compose.yml
index 4d6d839..3ba8cfa 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -66,7 +66,6 @@ x-services:
       - --dev-fast-reconnect
       - --plugin=/root/hold
       - --plugin=/root/clnurl
-      - --plugin=/root/mpay.sh
       - --experimental-offers
       - --clnurl-host=0.0.0.0
       - --ignore-fee-limits=false
