diff --git a/data/backend/boltz.conf b/data/backend/boltz.conf
index 6129eba..ccc3014 100755
--- a/data/backend/boltz.conf
+++ b/data/backend/boltz.conf
@@ -59,11 +59,11 @@ minSwapAmount = 100
   minSwapAmount = 25_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 400
+  reverse = 400
+  swapTaproot = 400
+  swapMinimal = 400
+  swapMaximal = 450
 
 [[pairs]]
 base = "RBTC"
