diff --git a/data/backend/boltz.conf b/data/backend/boltz.conf
index a07ff9d..9461291 100755
--- a/data/backend/boltz.conf
+++ b/data/backend/boltz.conf
@@ -39,11 +39,11 @@ maxSwapAmount = 40_294_967
 minSwapAmount = 50_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 180
+  reverse = 180
+  swapTaproot = 450
+  swapMinimal = 400
+  swapMaximal = 450
 
 [[pairs]]
 base = "L-BTC"
@@ -62,11 +62,11 @@ minSwapAmount = 100
   minSwapAmount = 25_000
 
   [pairs.timeoutDelta]
-  chain = 1440
-  reverse = 1440
-  swapMinimal = 1440
-  swapMaximal = 2880
-  swapTaproot = 10080
+  chain = 180
+  reverse = 180
+  swapTaproot = 450
+  swapMinimal = 400
+  swapMaximal = 450
 
 [[pairs]]
 base = "RBTC"
